templates/veoibd.yaml

```diff
@@ -57,6 +57,7 @@ Resources:
     Properties:
       ManagedPolicyArns:
         - arn:aws:iam::aws:policy/service-role/AWSBatchServiceRole
+        - !GetAtt PublishToCloudwatchPolicy.Arn
```

Comment:
this need to be `!Ref PublishToCloudwatchPolicy`

From https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-managedpolicy.html
```
Ref

When you pass the logical ID of this resource to the intrinsic Ref function, Ref returns the ARN.

In the following sample, the Ref function returns the ARN of the CreateTestDBPolicy managed policy, such as arn:aws:iam::123456789012:policy/teststack-CreateTestDBPolicy-16M23YE3CS700.
```